istioctl/pkg/writer/envoy/configdump/configdump.go

```diff
@@ -65,7 +65,7 @@ func (c *ConfigWriter) PrintBootstrapDump(outputFormat string) error {
 	}
 	out, err := protomarshal.ToJSONWithIndent(bootstrapDump, "    ")
 	if err != nil {
-		return fmt.Errorf("unable to marshal bootstrap in Envoy config dump")
+		return fmt.Errorf("unable to marshal bootstrap in Envoy config dump: %w", err)
```

Comment:
I think here can be `%v`?

Reply:
BTW looks like this can be added with an error too?

https://github.com/istio/istio/pull/48573/files#diff-1bb95541aafa1dcfa86778bd78df753f9f2162b2f7e7a02edbb65208adb7d6c3R92

Reply:
> I think here can be `%v`?

For application, i default to wrap the origin error.
For lib/sdk, i don't wrap by default.

In istio, we should prefre to dont wrap?

Reply:
Not sure about the security part but I think for this istioctl command, since the error will be returned and reported, there's no need to wrap?

Reply:
I change it to %v